CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction initially and potentially save a lot of time and effort.
+Also somebody can already start discussing or working on implementing the same or similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be worth, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly.
+
+We assume you use Linux as a work environment (you can use a virtual machine as well).
+
+1. Install ``docker``, launch it.
+
+2. Clone the [ansible-core](https://github.com/ansible/ansible) repository:
+```bash
+git clone https://github.com/ansible/ansible.git
+```
+3. Go to the cloned repository and prepare the environment:
+```bash
+cd ansible && source hacking/env-setup
+cd ~
+```
+
+4. Fork the ``community.mysql`` repository via the GitHub web interface.
+
+5. Clone the forked repository from your profile:
+```bash
+git clone https://github.com/YOURACC/community.mysql.git
+```
+
+6. Create the following directories in your home directory:
+```bash
+mkdir -p ~/ansible_collections/community
+```
+
+7. Move the cloned repository:
+```bash
+mv community.mysql ~/ansible_collections/community/mysql
+```
+
+8. Go there:
+```bash
+cd ~/ansible_collections/community/mysql
+```
+
+9. Be sure you are in the main branch:
+```bash
+git status
+```
+
+10. Show remotes. There should be the ``origin`` repository only:
+```bash
+git remote -v
+```
+
+11. Add the ``upstream`` repository:
+```bash
+git remote add upstream https://github.com/ansible-collections/community.mysql.git
+```
+
+12. Update your local ``main`` branch:
+```bash
+git fetch upstream
+git merge upstream/main
+```
+
+13. Create a branch for your changes:
+```bash
+git branch -b name_of_my_branch
+```
+
+14. In any case, a good approach is to start from writing [integration tests](https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html).
+
+Briefly, all integration tests are just Ansible roles and are stored in ``tests/integration/targets`` subdirectories.
```

Comment:
I would move this big chunk of text into another section, so one can complete the steps without having to read it.

Reply:
Good spot!

Reply:
Done + I simplified it and add a bit of useful information.